### DISCUSSION TITLE
Option for scrollback or history length

### DISCUSSION THREAD

**Discussion by Alb11747** [Chatterino/chatterino2#3617] [Ideas] [answered]
A new setting can be introduced that allows the user to change how many messages the chat holds at a time.
From what I was able to understand, the scrollback length for channels is currently a static 1000 in "src/messages/LimitedQueue.hpp".
This feature would immiate FFZ:
![image](https://user-images.githubusercontent.com/40252880/159197515-1b1cc853-f000-4a32-b2e8-10e248acaea5.png)

**Comment by Felanbird** [comment, marked as answer]
#11
